cn.navclub.ldbfx.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I mentionned this in the PR you close and thus it got ignored. There is an appdata file in the upstream repo that is installed. So you don't need that one here.

It's never installed anyway.

Reply:
OK, I'll remove the appdata